README.md

```diff
@@ -306,6 +306,26 @@ The content within these files can be accessed as such:
 {{> form-toggle}}
 ```
 
+#### Ordering
+
+You can manually order materials by prefixing the file name with numbers:
+
+```
+01-foo.html
+01.01-bar.html
+02-qux.html
+```
+
+This defines the order in which materials will appear in the side menu or other places the `materials.items` context is used.
+
```

Comment:
Would it be useful to update the `README` somewhere around here to mention that the `name` property for materials will also reflect the prefix-stripping? (I mean, I assume the `name` property is also cleaned up).